cfgov/v1/wagtail_hooks.py

```diff
@@ -245,6 +246,27 @@ def serve_latest_draft_page(page, request, args, kwargs):
         return response
 
 
+@hooks.register('register_reports_menu_item')
+def register_unpublished_changes_report_menu_item():
+    return AdminOnlyMenuItem(
+        "Page metadata",
```

Comment:
For consistency with the existing "Locked Pages" menu item title-case capitalization.

```suggestion
        "Page Metadata",
```